test/unit/mongodb-legacy.test.ts

```diff
@@ -0,0 +1,9 @@
+import { expect } from 'chai';
+
+import { Db } from '../mongodb';
```

Comment:
from slack, test more things, and move the file into `tools`

Reply:
done